csharp/src/Microsoft.AI.MachineLearning.Interop/Microsoft.AI.MachineLearning.targets

```diff
@@ -9,9 +9,11 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <WindowsAIBinary>$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\native\Microsoft.AI.MachineLearning.dll</WindowsAIBinary>
-    <OnnxRuntimeBinary>$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\native\onnxruntime.dll</OnnxRuntimeBinary>
-    <DirectMLBinary>$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\native\directml.dll</DirectMLBinary>
+    <RuntimesDirectory Condition="'$(OutputType)' == 'AppContainerExe'">$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\uap</RuntimesDirectory>
```

Comment:
>AppContainerExe [](start = 54, length = 15)

What's AppContainerExe?

Reply:
Executable built to run in an appcontainer (the store container isolation)